Assignment/solution_student/index.html

```diff
@@ -15,17 +15,69 @@
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <div id="header">
+        <h1>Relaxr</h1>
+        <nav>
+          <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">FAQ</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Blog</a></li>
+          </ul>
+        </nav>
+      </div>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+      <div id="main">
+        <article>
```

Comment:
The way you have it marked up right now, both blog posts are contained in the same `article` tag, but they are each definitely distinct articles, so I would separate the two blog posts into their own `article` tags. The way you've styled this is going need to change if you do this though because right now you are floating the `article` tag to the left and giving it the `60%` width etc., so your layout will break if you break the posts up into separate articles, but that's okay, I think it should be restructured anyway. What I would do is move the closing tag of your `#main` container, which is currently wrapping both the articles and the `#sidebar`, up so that it only contains the articles — then you will have a `#main` container, and a separate `#sidebar` container (which should include the ad unit in it as well), so you can float the `#main` container left at `60%` width, and so on. Of course, you'll need to wrap both the `#main` and `#sidebar` elements in a container with the `.clearfix` class so the floats don't cause issues with the following site footer elements. If that description was hard to follow, I put the revised code here for you: https://gist.github.com/poopsplat/3ba18941fa1b9aabf605 + https://gist.github.com/poopsplat/e637a559f5c772ceeb52